ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -40,6 +40,16 @@ const APP_FIELDS = [
 const APP_LIST_FIELDS = ['metadata.resourceVersion', ...APP_FIELDS.map(field => `items.${field}`)];
 const APP_WATCH_FIELDS = ['result.type', ...APP_FIELDS.map(field => `result.application.${field}`)];
 
+document.onkeypress = e => {
+    if (e.keyCode === 47) {
+        const searchBar = document.getElementById('search-bar');
```

Comment:
Same as https://github.com/argoproj/argo-cd/pull/4801#discussion_r520898463 . Please use [React.useRef](https://reactjs.org/docs/hooks-reference.html#useref) instead of `document.getElementById('search-bar')`